taxonomies/categories.txt

```diff
@@ -25798,8 +25888,15 @@ nl:Gruyère uit Frankrijk, Franse Gruyère
 country:en:France
 wikidata:en:Q3118260
 
+
+<en:Cow cheeses
+<en:Uncooked pressed cheeses
 <en:French cheeses
-fr:Laguiole
+en:Cantal Salers or Laguiole cheese from cow's milk
+fr:Cantal Salers, Laguiole
```

Comment:
Those are different cheeses, we already have categories for them, we need to apply the ciqual codes to the existing categories.

Reply:
Cantal has a slightly different CIQUAL code